packages/generators/generators/lib/files/ts/plugin/server/routes/admin/index.ts

```diff
@@ -0,0 +1,6 @@
+import myRoute from './my-route';
+
+export default {
+  routes: [myRoute],
```

Comment:
I think here you have an array with an array of objects and you just want an array of objects. Also `myRoutes`, plural might be better since it is an array of routes. Wdyt? If you agree could you make that change for the js files as well.

```suggestion
  routes: myRoute,
```